<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentView</class>
 <widget class="QWidget" name="StudentView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="PasswordBtn">
   <property name="geometry">
    <rect>
     <x>372</x>
     <y>430</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>更改密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordlineEdit">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>430</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>430</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入更改后的密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ReturnwinBtn">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>490</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addpictureBtn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>430</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>更改个人头像</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>50</y>
     <width>161</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="idlabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>55</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="sexlabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="namelabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="agelabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="majorlabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="classlabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="phonelabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="roomlabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bedlabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>380</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
